ssg/entities/profile_base.py

```diff
@@ -112,6 +113,9 @@ def _should_have_version(self):
             'version'] is not None
 
     def to_xml_element(self):
+        if self.hidden:
+            return ET.Comment('Default Profile')
```

Comment:
I think it shouldn't return a comment.

Reply:
So, here is the problem. It is expected that this function will return an `ET.Element` (the comment is actually `ET.Element(Comment)`).

This comment will be present in the `ssg-blah-xccdf.xml` file, but would be filtered out in the final `ssg-blah-ds.xml` data stream.

That still is the best idea I have about how to handle it.